migrations/versions/2e0d2e02a721_.py

```diff
@@ -0,0 +1,70 @@
+"""empty message
+
+Revision ID: 2e0d2e02a721
+Revises: 6b1274f61edd
+Create Date: 2022-09-23 11:03:38.783561
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+
+# revision identifiers, used by Alembic.
+revision = '2e0d2e02a721'
+down_revision = '6b1274f61edd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('kvm', 'timestamp_build_finished')
+    op.alter_column('regression_test', 'expected_rc',
+               existing_type=mysql.INTEGER(),
+               nullable=True,
+               existing_server_default=sa.text("'0'"))
+    op.drop_constraint('regression_test_ibfk_1', 'regression_test', type_='foreignkey')
+    op.create_foreign_key('regression_test_fk', 'regression_test', 'sample', ['sample_id'], ['id'], onupdate='CASCADE', ondelete='CASCADE')
+    op.alter_column('test_result', 'runtime',
+               existing_type=mysql.INTEGER(),
+               nullable=True)
+    op.alter_column('test_result', 'exit_code',
+               existing_type=mysql.INTEGER(),
+               nullable=True)
+    op.alter_column('test_result', 'expected_rc',
+               existing_type=mysql.INTEGER(),
+               nullable=True,
+               existing_server_default=sa.text("'0'"))
+    op.drop_constraint('upload_ibfk_1', 'upload', type_='foreignkey')
+    op.create_foreign_key('upload_fk', 'upload', 'user', ['user_id'], ['id'], onupdate='CASCADE', ondelete='RESTRICT')
```

Comment:
@thealphadollar @canihavesomecoffee These changes were present already? (although the migration file for this was never uploaded)